distribution/docs/src/main/resources/_contents/_configuring/configuring-from-configuration-files-contents.adoc

```diff
@@ -0,0 +1,746 @@
+
+As most configurations are stored in configuration files, in some instances it may make sense to edit those configuration files directly.
```

Comment:
The other main use case is to use pre-created configuration files to configure the system (see section 3.3.2, 3rd paragraph). We may want to mention this here as well.